ts/parts-map/HeatmapSeries.ts

```diff
@@ -514,7 +606,47 @@ seriesType<Highcharts.HeatmapSeriesOptions>(
 
         /* eslint-enable valid-jsdoc */
 
-    }, colorPointMixin)
+    },
+    colorPointMixin,
+    {
+        setState: function (
+            this: Highcharts.HeatmapPoint,
+            state?: string
+        ): void {
+            var point = this,
+                series = this.series,
+                points = series.points,
+                inactiveException = (series.options.states as any)
+                    .inactive.except as Highcharts
+                    .HeatmapSeriesStatesInactiveExceptValue,
+                filterFunction = series.inactiveFilters[inactiveException],
+                i = 0;
+
+            if (filterFunction) {
+                series.options.inactiveOtherPoints = true;
+
+                for (; i < points.length; i++) {
+                    // Check if points are not being destroyed
+                    if (points[i] && points[i].series) {
+                        colorPointMixin.setState.call(
+                            points[i],
+                            state !== 'hover' ||
+                            (
+                                state === 'hover' &&
+                                filterFunction(point, points[i])
+                            ) ?
+                                '' : 'inactive'
+                        );
+                    }
+                }
+
+                series.options.inactiveOtherPoints = true;
+            }
+
+            // Apply default state for current point:
+            colorPointMixin.setState.apply(this, arguments as any);
```

Comment:
Can be avoided with `[].from(arguments)`

Reply:
I don't see anywhere this usage. Is it a planned change in future for all `arguments as any` -> `[].from(arguments)` (reminder: no IE support). 


Anyway, changing this throws errors in `npx gulp script`:
> ts/parts-map/HeatmapSeries.ts(647,53): error TS2339: Property 'from' does not exist on type 'never[]'.

Reply:
Well the `any` type are turning off all TypeScript checks and therefor have to be removed in the future. How we will do this is not clear yet.